### ISSUE TITLE
SqlServer script is executed as batch even if it does not use the GO keyword

### ISSUE THREAD

**Issue by orck-adrouin** [fluentmigrator/fluentmigrator#842]
Hello,

I have a script that is executed as a "batch" instead of an ordinary query.  Usually it is not important however in this case I am missing empty lines in my data because the batch processing is stripping empty lines.

```sql
insert into MyTable (Id, Data)
values (42, 'This is a list of games played by people

DooM
Pokemon GO
Potato-o-matic');
```
In this sample code, the line above DooM is lost.

My understanding of the code is that this issue is caused by line [156 of SqlServerProcessor.cs](https://github.com/fluentmigrator/fluentmigrator/blob/f17dca3ba0ada883bf016d7bf297c9e0b8b8bdf7/src/FluentMigrator.Runner/Processors/SqlServer/SqlServerProcessor.cs#L156) .  The code is looking for the letters "GO" anywhere in the script instead of looking for a "GO" on its own line.  The logic at line 201 of the same file should be applied for the batch detection.

**Comment by fubar-coder** [comment]
I'm currently trying to implement an [SQL batch parser](https://github.com/fluentmigrator/batch-parser) which should solve this problem.

**Comment by fubar-coder** [comment]
Fixed in release/2.0.0
